### PR TITLE
Bump controller-runtime to v0.9.7

### PR DESCRIPTION
**Pull request by adammw**
Bump controller-runtime to v0.9.7 and its dependencies.

Needed this change locally to be able to setup/run tests correctly.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,11 +3,12 @@ module github.com/open-policy-agent/cert-controller
...
+	github.com/onsi/gomega v1.15.0
 	github.com/pkg/errors v0.9.1
-	go.uber.org/atomic v1.6.0
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
-	sigs.k8s.io/controller-runtime v0.8.2
+	go.uber.org/atomic v1.7.0
+	k8s.io/api v0.21.4
+	k8s.io/apiextensions-apiserver v0.21.4
```
Comment (sozercan) [inline go.mod:10]: `go mod tidy` removes this line for me. Looks like you added part of #37?

**Review by sozercan — APPROVED** [review]
LGTM pending `apiextensions-apiserver` line

File: go.mod (hunk 1 above)
Comment (adammw) [reply go.mod]: yep, fixed.

**Review by maxsmythe — APPROVED** [review]
LGTM

**Pull request merged**